src/core/namespace.c

```diff
@@ -1623,9 +1624,11 @@ static int mount_image(
                 return 0;
         if (r == -ESTALE && rdata.os_release_id)
                 return log_error_errno(r, // FIXME: this should not be logged ad LOG_ERR, as it will result in duplicate logging.
-                                       "Failed to mount image %s, extension-release metadata does not match the lower layer's: ID=%s%s%s%s%s%s%s",
+                                       "Failed to mount image %s, extension-release metadata does not match the lower layer's: ID=%s%s%s%s%s%s%s%s%s",
                                        mount_entry_source(m),
                                        rdata.os_release_id,
+                                       rdata.os_release_id_like ? " ID_LIKE=" : "",
+                                       strempty(rdata.os_release_id_like),
```

Comment:
Hmm, I think we need to wrap the rhs in quotes. If there are spaces in the field, this message will be confusing.
But then it's fine to drop the conditionalization, `ID_LIKE=''` is clear too.